app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/topics.controllers");
+const { psqlErrors, customErrors, serverErrors } = require("./errors");
+const app = express()
+
+app.use(express.json())
+
+function getApi (req, res) {
+    res.status(200).send({ msg : 'all okay'})
+};
+app.get("/api/healthcheck", getApi);
```

Comment:
While I understand why you may of included these to get a start they aren't really needed or in the scope of this sprint.